kf_lib_data_ingest/etl/extract/utils.py

```diff
@@ -76,20 +76,60 @@ def _obvert_visibility(self, df):
         vice versa.
         """
 
+        def x_to_bool_or_none(x):
+            # Convert all obvious values to True/False/None and raise ValueError
+            # for any special cases, which are no longer allowed here.
+            if isinstance(x, bool):
+                return x
+            elif pandas.isna(x):
```

Comment:
```suggestion
            elif pandas.isnull(x):
```
Pandas says `isnull` also includes NaT (null datetime-like)? Other than that, idk what the difference is

Reply:
I don't see any difference between
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.isna.html
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.isnull.html

Reply:
Also https://github.com/pandas-dev/pandas/blob/73c68257545b5f8530b7044f56647bd2db92e2ba/pandas/core/dtypes/missing.py#L141